server/models/users.js

```diff
@@ -86,13 +86,19 @@ export default (sequelize, Sequelize) => {
     User.hasMany(db["Favourites"], {
       foreignKey: "userId",
       target: "id",
-      onUpdate: "CASCADE"
+      onUpdate: "CASCADE",
+      onDelete: "CASCADE"
     });
     User.hasMany(db["Articles"], {
       foreignKey: "authorId",
       target: "id",
       onDelete: "CASCADE"
     });
+    User.hasMany(db["Comments"], {
+      foreignKey: "userId",
+      target: "id",
+      onDelete: "cascade"
```

Comment:
Please be consistent. Please change `cascade` to `CASCADE`

Reply:
Okay, I will implement change.

Reply:
The change has been implemented